astronomer/providers/amazon/aws/example_dags/example_emr_eks_containers_job.py

```diff
@@ -0,0 +1,155 @@
+    "EKS_CONTAINER_PROVIDER_CLUSTER_NAME", "providers-team-eks-cluster"
+)
+KUBECTL_CLUSTER_NAME = os.getenv("KUBECTL_CLUSTER_NAME", "providers-team-eks-namespace")
+VIRTUAL_CLUSTER_NAME = os.getenv("EMR_VIRTUAL_CLUSTER_NAME", "providers-team-virtual-eks-cluster")
+AWS_ACCESS_KEY_ID = os.getenv("AWS_ACCESS_KEY_ID", "xxxxxxx")
+AWS_SECRET_ACCESS_KEY = os.getenv("AWS_SECRET_ACCESS_KEY", "xxxxxxxx")
+AWS_DEFAULT_REGION = os.getenv("AWS_DEFAULT_REGION", "us-east-2")
+
+default_args = {
+    "execution_timeout": timedelta(minutes=30),
+}
+
+
+def create_emr_virtual_cluster_func():
+    """Create EMR virtual cluster in container"""
+    client = boto3.client("emr-containers")
+    try:
+        response = client.create_virtual_cluster(
+            name=VIRTUAL_CLUSTER_NAME,
+            containerProvider={
+                "id": EKS_CONTAINER_PROVIDER_CLUSTER_NAME,
+                "type": "EKS",
+                "info": {"eksInfo": {"namespace": KUBECTL_CLUSTER_NAME}},
+            },
+        )
+        os.environ["VIRTUAL_CLUSTER_ID"] = response["id"]
+    except ClientError:
+        logging.exception("Error while creating EMR virtual cluster")
+        return None
+
+
+# [START howto_operator_emr_eks_config]
+JOB_DRIVER_ARG = {
+    "sparkSubmitJobDriver": {
+        "entryPoint": "local:///usr/lib/spark/examples/src/main/python/pi.py",
+        "sparkSubmitParameters": "--conf spark.executors.instances=2 --conf spark.executors.memory=2G --conf spark.executor.cores=2 --conf spark.driver.cores=1",  # noqa: E501
+    }
+}
+
+CONFIGURATION_OVERRIDES_ARG = {
+    "applicationConfiguration": [
+        {
+            "classification": "spark-defaults",
+            "properties": {
+                "spark.hadoop.hive.metastore.client.factory.class": "com.amazonaws.glue.catalog.metastore.AWSGlueDataCatalogHiveClientFactory",  # noqa: E501
+            },
+        }
+    ],
+    "monitoringConfiguration": {
+        "cloudWatchMonitoringConfiguration": {
+            "logGroupName": "/aws/emr-eks-spark",
+            "logStreamNamePrefix": "airflow",
+        }
+    },
+}
+# [END howto_operator_emr_eks_config]
+
+with DAG(
+    dag_id="emr_eks_pi_job",
+    start_date=datetime(2022, 1, 1),
+    schedule_interval=None,
+    catchup=False,
+    default_args=default_args,
+    tags=["emr_containers", "example"],
+) as dag:
+    # Task steps for DAG to be self-sufficient
+    setup_aws_config = BashOperator(
+        task_id="setup_aws_config",
+        bash_command=f"aws configure set aws_access_key_id {AWS_ACCESS_KEY_ID}; "
+        f"aws configure set aws_secret_access_key {AWS_SECRET_ACCESS_KEY}; "
+        f"aws configure set default.region {AWS_DEFAULT_REGION}; ",
+    )
+
+    # Task to create EMR clusters on EKS
+    create_EKS_cluster_kube_namespace_with_role = BashOperator(
+        task_id="create_EKS_cluster_kube_namespace_with_role",
+        bash_command="sh /usr/local/airflow/dags/example_create_emr_on_eks_cluster.sh ",
```

Comment:
script path is hardcoded can we change it to "sh $AIRFLOW_HOME/dags/example_create_emr_on_eks_cluster.sh"

Reply:
Done